persistence/src/main/scala/za/co/absa/spline/persistence/ArangoConnectionURL.scala

```diff
@@ -17,23 +17,24 @@
 package za.co.absa.spline.persistence
 
 import org.apache.commons.lang3.StringUtils.trimToNull
-import za.co.absa.commons.lang.OptionImplicits.StringWrapper
 import za.co.absa.spline.persistence.ArangoConnectionURL.ArangoSecureDbScheme
 
 import java.net.MalformedURLException
+import scala.collection.mutable
 import scala.util.matching.Regex
 
 case class ArangoConnectionURL(scheme: String, user: Option[String], password: Option[String], hosts: Seq[(String, Int)], dbName: String) {
 
-  import za.co.absa.commons.lang.OptionImplicits._
+  import za.co.absa.commons.lang.extensions.AnyExtension._
+  import za.co.absa.commons.lang.extensions.StringExtension._
 
   require(user.isDefined || password.isEmpty, "user cannot be blank if password is specified")
 
   def asString: String = {
     val userInfo = trimToNull(Seq(user, password.map(_ => "*****")).flatten.mkString(":"))
     val commaSeparatedHostsString = hosts.map { case (host, port) => s"$host:$port" }.mkString(",")
 
-    new StringBuilder()
+    new mutable.StringBuilder()
```

Comment:
Isn't it the same Class?

Reply:
It is - `scala.collection.mutable.StringBuilder`
Scala guide recommends to explicitly use `mutable` package name to clearly distinguish between mutable and immutable classes.